.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,72 @@
+name: Bug Report
+description: Report a bug encountered while operating sealer
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Environment
+  - type: input
+    id: version
+    attributes:
+      label: What is the version of Sealer you using?
+      description: use `sealer version`
+      placeholder: ex. {"gitVersion":"v0.8.5","gitCommit":"f9c3d99","buildDate":"2022-04-28 14:16:58","goVersion":"go1.16.15","compiler":"gc","platform":"linux/amd64"}
+    validations:
+      required: true
+  - type: input
+    id: os
+    attributes:
+      label: What is your OS environment?
+      description: e.g `cat /etc/os-release`
+      placeholder: ex. Ubuntu 16.04
+    validations:
+      required: true
+  - type: input
+    id: kernel
+    attributes:
+      label: What is the Kernel version?
+      description: e.g `uname -a`
+      placeholder: ex. Linux sealer 5.4.0-113-generic
+    validations:
+      required: false
+  - type: textarea
+    id: other-env
+    attributes:
+      label: Other environment you want to tell us?
+      value: |
+        - Cloud provider or hardware configuration:
+        - Install tools:
+        - Others:
+  - type: markdown
+    attributes:
+      value: |
+        # Problem
+  - type: textarea
+    id: wrong
+    attributes:
+      label: What happen?
+      description: describe the problem you meet.
```

Comment:
Could you put this problem description at the top of the issue? The sealer, os, distribution version could put at the bottom part of the issue.